controller/space.go

```diff
@@ -546,6 +547,18 @@ func (c *SpaceController) Update(ctx *app.UpdateSpaceContext) error {
 	return ctx.OK(&response)
 }
 
+const (
+	// see https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/identifiers.md
+	spaceNameMaxLength int    = 63
```

Comment:
@xcoulon space_max_length is hard coded at [link](https://github.com/fabric8-services/fabric8-wit/pull/2100/files#diff-5147d84da6b10c041a0bafd5935f9f4fR105).  See if you can have shared variable here.

Reply:
yes, but I remove the use of the `nameValidationFunction` at the design level to validate the space name, actually. I'm not sure why the same length rule applies to `area`, etc., though...